terraform/README.md

```diff
@@ -1,31 +1,52 @@
+* Use the Concourse pipeline to provision it in an automated manner. That pipeline uses the terraform configurations 
+found in this directory.  
 
-## Config
-The following config is required to deploy the project:
-- release_bucket_name: `Optional` Release bucket for payment-reconciler, defaults to `release.ch.gov.uk`
-- version: `Required` Github release version
-- stage: `Required` deployment stage
-- environment: `Required` to read the environment variable from.
+## Configuration
 
-## Cheat Sheet
-tested on terraform version `0.11.7`, intended use with concourse. There should be no need for a manual deployment, instead add a specific environment to the concourse pipeline.
+To deploy the reconciler, we must specify the following configuration properties:
 
-```sh
-terraform get
+- environment: `Required` The environment to be deployed to 
+- aws_profile: `Required` The AWS profile
+- cron_schedule: `Required` A cron schedule for the triggering of the lambda
+- aws_region: `Required` The AWS region
+- release_bucket_name `Required` The name of the release bucket containing the distribution 
+- remote_state_bucket: `Required` The S3 bucket where the terraform state for the network is stored
+- remote_state_key: `Required` The location in the S3 bucket where the terraform state of the network components is read
+ from
 
-source ./environments/development/dev/terraform.cfg
+## Manual deployment
 
-#initialize terraform
-terraform init 
+In addition to the above properties, for a manual terraform deployment, use 
+[platform-tools](https://github.com/companieshouse/platform-tools). The following arguments must be 
+provided on the command line as in the example below:
 
-# Plan provisioning
-TF_VAR_version=<release number> terraform plan -var-file=environments/development/<environment>/vars
-
-# Apply
-TF_VAR_version=<release number> terraform apply -var-file=environments/development/<environment>/vars
 ```
+terraform-runner -g lambda -c plan -p development-eu-west-2 -e cidev
+```
+
+The following inputs will be required for terraform too:
+
+- var.release_version: `Required` The Github release version to be deployed
+- var.vault_password: `Required` The password required for access to the terraform vault for other configuration parameters
+- var.vault_username: `Required` The user required for access to the terraform vault
+- provider.vault.address: `Required` The address of the terraform vault
+
+### Environment variables to reduce repetition
+
+To avoid having to enter the same values over and over again when testing the provisioning of the reconciler, use the 
+following environment variables: 
+
+| Variable name | Equivalent to | Example |
+| ------------- | ---------------- | ------- |
+| TF_VAR_vault_password | var.vault_password | export TF_VAR_vault_password=XXXXXXXXX |
+| TF_VAR_vault_username | var.vault_username | export TF_VAR_vault_username=payment_reconciler_read |
```

Comment:
Again, you shouldn't be putting specifics in here. This is a public repo

Reply:
Will replace with <USER NAME>.